src/output/tts/tts_offline.py

```diff
@@ -2,10 +2,14 @@
 
 
 class TtsOffline:
+    """
+    TTS implementation which uses the library `pyttsx3` which works fully offline
+    """
+
     def __init__(self):
         # initialize Text-to-speech engine
```

Comment:
shouldn't we change this comment to dogstring ? is it even necessary?

Reply:
it's indeed unnecessary. i've removed it